src/sentry/demo/data_population.py

```diff
@@ -93,60 +86,57 @@ def get_event_from_file(file_name):
         return clean_event(json.load(f))
 
 
-def distribution_v1(hour: int) -> int:
-    if hour > 9 and hour < 12:
-        return 8
-    if hour > 6 and hour < 15:
-        return 3
-    if hour > 4 and hour < 20:
-        return 2
-    return 1
+def distribution_v1() -> list:
```

Comment:
@roggenkemper why switch the distribution functions like this? Cant' say I'm a huge fan cause it makes it harder to make new functions IMO

Reply:
Keeping it as a function gives us flexibility. If you want to use arrays, I'd suggest something ike:

```
def distribution_v1(hour: int) -> int:
   dist = [1, 3, 1, 2, 1, 1, 1, 2, 4, 1, 2, 3, 4, 5, 4, 3, 4, 7, 1, 2, 2, 2, 1, 3]
   return dist[hour]
```

Reply:
I had changed it to make it easier to visualize when I was having issues with the number of sessions being created. will change it back!

Reply:
@roggenkemper makes sense! See my 2nd comment

Reply:
since the issue ended up being more about scaling than anything else, I think keeping it as if statements makes sense!